aioservertiming/__init__.py

```diff
@@ -54,7 +56,7 @@ def __repr__(self):
 async def server_timing_mware(
     request: web.Request,
     handler: Handler
-) -> web.Response:
+) -> web.StreamResponse:

```

Comment:
Not sure about this one, PyCharm suggested this change

Reply:
I think web.Response is more suitable here.